tests/vmi_lifecycle_test.go

```diff
@@ -1033,10 +1026,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				config.CPUModel = defaultCPUModel
 				tests.UpdateKubeVirtConfigValueAndWait(*config)
 
-				newVMI := newCirrosVMI()
-				newVMI.Spec.Domain.CPU = &v1.CPU{
-					Model: vmiCPUModel,
-				}
+				newVMI := libvmi.NewCirros(libvmi.WithCPU(&v1.CPU{Model: vmiCPUModel}))
```

Comment:
I think this is the right thing to do.
Just note that recently `WithCPUModel` was added which does the same.